apps/coscrad-frontend-e2e/src/e2e/resources/videos/video.index.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,204 @@
+import { AggregateType, LanguageCode } from '@coscrad/api-interfaces';
+
+const dummyVideoTitle = 'Dinosaur Video Title';
+
+const buildAggregateCompositeIdentifier = (id: string) => ({
+    type: AggregateType.video,
+    id: `9b1deb4d-3b7d-4bad-9bdd-2b0d7b100${id}`,
+});
+
+const aggregateCompositeIdentifier = buildAggregateCompositeIdentifier('001');
+
+const validUrl =
+    'https://coscrad.org/wp-content/uploads/2023/05/Rexy-and-The-Egg-_3D-Dinosaur-Animation_-_-3D-Animation-_-Maya-3D.mp4';
+
+// TODO[https://www.pivotaltracker.com/story/show/185717928] seed data with a title translation and unskip this test
+describe.skip(`Video Index-to-detail Query Flow`, () => {
+    before(() => {
+        cy.clearDatabase();
+
+        cy.seedTestUuids(100);
+
+        cy.executeCommandStreamByName('users:create-admin');
+
+        cy.seedDataWithCommand(`CREATE_VIDEO`, {
+            title: dummyVideoTitle,
+            languageCodeForTitle: LanguageCode.Chilcotin,
+            // titleEnglish: dummyEnglishTranslationOfVideoTitle,
+            aggregateCompositeIdentifier,
+            audioURL: validUrl,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier,
+        });
+    });
+
+    describe(`the index page`, () => {
+        it.skip(`should display the label "Videos"`, () => {
+            cy.contains(`Videos`);
+        });
+
+        describe(`when there is a single video`, () => {
+            beforeEach(() => {
+                cy.visit('/Resources/Videos');
+            });
+
+            describe(`when the video has a title in both languages`, () => {
+                it(`should display the title in both languages`, () => {
+                    cy.contains(dummyVideoTitle);
+
+                    // TODO[https://www.pivotaltracker.com/story/show/185717928] seed data with a title translation
+                    // cy.contains(dummyEnglishTranslationOfVideoTitle);
+                });
+            });
+        });
+
+        describe(`the search filter`, () => {
+            const videoTitleWithQDash = `Q-Bert's Jam`;
+
+            const searchScopes = [`allProperties`, `name`];
```

Comment:
If you're not exposing a 'has transcript' filterable property, you might want to consider that. Some people may be specifically looking for transcripts when they view videos.